internal/handler/interfaces.go

```diff
@@ -0,0 +1,21 @@
+package handler
+
+import (
+	"context"
+	"io/fs"
+
+	"helm.sh/helm/v3/pkg/chart"
+	"helm.sh/helm/v3/pkg/chartutil"
+
+	bd "github.com/operator-framework/operator-controller/internal/bundledeployment"
+)
+
+type Handler interface {
```

Comment:
Should this maybe become less generic, so `RegistryHandler` perhaps?

Reply:
Yes, there's a good argument for that since we're only using one of the 3 types from rukpak.